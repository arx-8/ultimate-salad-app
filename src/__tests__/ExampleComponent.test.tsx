import { render } from "@testing-library/react-native"
import { ExampleComponent } from "./ExampleComponent"

describe("renders correctly", () => {
  it("case 1", () => {
    const rendered = render(<ExampleComponent content="hello react-native" />)
    expect(rendered.toJSON()).toMatchSnapshot()
  })

  it("case 2", () => {
    const rendered = render(<ExampleComponent content="hello snapshot" />)
    expect(rendered.toJSON()).toMatchSnapshot()
  })
})
