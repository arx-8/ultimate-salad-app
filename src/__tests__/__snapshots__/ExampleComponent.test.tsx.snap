// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly case 1 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <Text
    style={
      Object {
        "fontSize": 20,
        "margin": 10,
        "textAlign": "center",
      }
    }
  >
    Welcome to React Native!
  </Text>
  <Text>
    This is a React Native snapshot test. 
    hello react-native
  </Text>
</View>
`;

exports[`renders correctly case 2 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <Text
    style={
      Object {
        "fontSize": 20,
        "margin": 10,
        "textAlign": "center",
      }
    }
  >
    Welcome to React Native!
  </Text>
  <Text>
    This is a React Native snapshot test. 
    hello snapshot
  </Text>
</View>
`;
